### ISSUE TITLE
Server-Side config option

### ISSUE THREAD

**Issue by OsoaGitHub** [tratteo/LiterallyChunkLoader-1.15#7]
This would make the mod compatible without needing it installed on the client side. (just to preface this I have no idea how fabric mods work, but everything I say might theoretically work). Instead of a custom texture and model, the block would have the texture of a block already existing in the vanilla game (lodestone, structure block.) On right clicking the block, instead of showing the modded GUI, it shows a chest with red stained glass representing the deactivated chunks and green stained glass representing the activated chunks. Clicking on the glass will toggle the chunks on / off.
Thanks for the cool mod

**Comment by nthnlws** [comment]
Good idea. VanillaTweaks does have a chunk loader as a data pack that uses a loadstone (of course you can edit that). It does only load the chunk that it is in though, and not any surrounding chunks.
